solution/backend/regcore/models.py

```diff
@@ -4,35 +4,64 @@
 from solo.models import SingletonModel
 
 
+class Title(models.Model):
+    name = models.CharField(max_length=8, unique=True)
+    last_updated = models.DateTimeField(auto_now=True)
+    toc = models.JSONField()
+
+    class Meta:
+        ordering = ("name",)
+
+
 class PartQuerySet(models.QuerySet):
     def effective(self, date):
         return self.filter(date__lte=date).order_by("name", "-date").distinct("name")
 
     def versions(self, title, part):
         return self.filter(name=part, title=title).order_by('-date').values("date")
 
+    def new_latest(self, title, part):  # TODO: rename to "latest"
+        query = self.filter(titleobject__name=title, name=part).order_by("-version")
+        return query[0] if len(query) > 0 else None
```

Comment:
Good call. Yes this will work after I can implement ordering of parts. Right now it would only return first by primary key which may not be correct. However I'm not actually using this right now so I can remove it.

Reply:
`new_latest` removed in [422eee0](https://github.com/CMSgov/cmcs-eregulations/pull/373/commits/422eee070101ac9b05443f6c8facd6a2650c99dc)